### ISSUE TITLE
Add that controller actions are supported for Ajax

### ISSUE THREAD

**Issue by DaWolfey** [themosis/documentation#82]
I discovered that you can use a controller@action with ajax::listen, this is not documented. Note that you have to specify the full namespace - \App\Http\Controllers\Controller@Action

**Comment by jlambe** [comment]
@DaWolfey That's correct, and this syntax works on the Page, Ajax, Actio and Filter APIs 😉
